sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CreateNewCaseSteps.java

```diff
@@ -172,6 +199,11 @@ private void fillDateOfReport(LocalDate date) {
     webDriverHelpers.fillInWebElement(DATE_OF_REPORT_INPUT, formatter.format(date));
   }
 
+  private void fillDateOfReportDE(LocalDate date) {
```

Comment:
we can keep the same method, just refactor it to pass a formatter for the desired market